### ISSUE TITLE
Running into errors during set up LLM serving

### ISSUE THREAD

**Issue by yidingjiang** [alpa-projects/alpa#781]
**Please describe the bug**
Hi! I am trying to set up LLM serving following the tutorial here: https://alpa.ai/tutorials/opt_serving.html
I was trying to run `python3 textgen.py --model facebook/opt-125m` but got the following error:
```
2022-11-20 22:31:28.919355: W external/org_tensorflow/tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-11-20 22:31:28.983202: W external/org_tensorflow/tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
A decoder-only architecture is being used, but right-padding was detected! For correct generation results, please set `padding_side='left'` when initializing the tokenizer.
Traceback (most recent call last):
  File "textgen.py", line 63, in <module>
    main(args)
  File "textgen.py", line 44, in main
    **generate_params)
  File "/home/yidingji/miniconda3/envs/opt/lib/python3.7/site-packages/torch/autograd/grad_mode.py", line 28, in decorate_context
    return func(*args, **kwargs)
  File "/home/yidingji/miniconda3/envs/opt/lib/python3.7/site-packages/transformers/generation_utils.py", line 1441, in generate
    if self.device.type != input_ids.device.type:
AttributeError: 'WrappedInferenceFunc' object has no attribute 'device'
```

**System information and environment**

OS Platform and Distribution (e.g., Linux Ubuntu 16.04, docker): CentOS Linux 7
Python version: Python 3.7.11
CUDA version: 11.1
NCCL version: 2708
cupy version: cupy-cuda111
GPU model and memory: GTX2080, 12G
Alpa version: 0.2.1
Pytorch version: 1.9.1+cu111
TensorFlow version: NA
JAX version: 0.3.15

**Comment by merrymercy** [comment]
Your transformers version is too high. As mentioned in our doc (https://alpa.ai/tutorials/opt_serving.html#requirements), could you downgrade it to <= 4.23.1?

**Comment by merrymercy** [comment]
Fixed by #784
